### DISCUSSION TITLE
Conflicts shown but not able to open in GitHub desktop

### DISCUSSION THREAD

**Discussion by alancoler** [community/community#106500] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have conflicts but unable to see them. When I click command line instructions and it shows using "git status" but when I use that I get "Your branch is up to date with 'origin/..."
How do I see the conflicts or resolve them?
<img width="931" alt="Screenshot 2024-02-06 at 11 11 32 AM" src="https://github.com/community/community/assets/12138500/1e230f20-21a7-4303-a989-beb08a8f630d">

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @alancoler ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by alancoler** [reply]
this is GitHub desktop, that is a GitHub product. ???

**Reply by queenofcorgis** [reply]
@alancoler yes it is, but this was posted in Discussions category which is just for conversations around the Discussions feature specifically 😄
